ACPs/204-precompile-secp256r1/README.md

```diff
@@ -0,0 +1,83 @@
+# ACP-204: Precompile for secp256r1 Curve Support
+
+| ACP | 204 |
+| :--- | :--- |
+| **Title** | Precompile for secp256r1 Curve Support |
+| **Author(s)** | [Santiago Cammi](https://github.com/scammi), [Arran Schlosberg](https://github.com/ARR4N)  |
+| **Status** | Proposed ([Discussion]()) |
+| **Track** | Standards |
+
+## Abstract
+
+This proposal introduces a precompiled contract that performs signature verifications for the secp256r1 elliptic curve on Avalanche's C-Chain. The precompile will be implemented at address `0x0200000000000000000000000000000000000006` and will enable native verification of P-256 signatures, significantly improving gas efficiency for biometric authentication systems, WebAuthn, and modern device-based signing mechanisms.
+
+## Motivation
+
+The secp256r1 (P-256) elliptic curve is the standard cryptographic curve used by modern device security systems, including Apple's Secure Enclave, Android Keystore, WebAuthn, and Passkeys. However, Avalanche currently only supports secp256k1 natively, forcing developers to use expensive Solidity-based verification that costs [200k-330k gas per signature verification](https://hackmd.io/@1ofB8klpQky-YoR5pmPXFQ/SJ0nuzD1T#Smart-Contract-Based-Verifiers).
+
+This ACP proposes implementing EIP-7212's secp256r1 precompiled contract to unlock significant ecosystem benefits:
+
+### **Enterprise & Institutional Adoption**
+- **Reduced onboarding friction**: Enterprises can leverage existing biometric authentication infrastructure instead of managing seed phrases or hardware wallets
+- **Regulatory compliance**: Institutions can utilize their approved device security standards and identity management systems
+- **Cost optimization**: 100x gas reduction (from 200k-330k to 3,450 gas) makes enterprise-scale applications economically viable
+
+The 100x gas cost reduction makes these use cases economically viable while maintaining the security properties institutions and users expect from their existing devices.
+
+## Specification
+
+This ACP implements [EIP-7212](https://eips.ethereum.org/EIPS/eip-7212) for secp256r1 signature verification on Avalanche. The specification follows EIP-7212 exactly, with the precompiled contract deployed at address `0x0200000000000000000000000000000000000006`.
```

Comment:
```suggestion
This ACP implements [EIP-7212](https://eips.ethereum.org/EIPS/eip-7212) for secp256r1 signature verification on Avalanche. The specification follows EIP-7212 exactly, with the precompiled contract deployed at address `0x0200000000000000000000000000000000000006`, imported to coreth via the precompile [registry](https://github.com/ava-labs/coreth/blob/master/precompile/registry/registry.go).
```
The registry's comments will include the [deployed address listed](https://github.com/ava-labs/coreth/pull/990/files) if the precompile is added.